backend/app/Http/Controllers/AccountController.php

```diff
@@ -2,7 +2,18 @@
 
 namespace App\Http\Controllers;
 
+use App\Http\Resources\AccountResource;
+use App\Models\Account;
+
 class AccountController extends Controller
 {
-    //
+    public function index()
+    {
+        return AccountResource::collection(Account::all());
+    }
+
+    public function show(Account $account)
+    {
+        return AccountResource::make(($account));
+    }
```

Comment:
was this checked if this was working?

wrong parameters for the show function
should just be $id

Reply:
![image](https://github.com/framgia/sph-flutter/assets/114897466/e9445f15-bd4b-4561-a62b-2f085ca8b11a)

Reply:
I've moved it out of the auth middleware to test but i don't think it affects it

Reply:
ok, just remove the double parenthesis